graphql/inputs.graphql

```diff
@@ -213,6 +213,7 @@ input ListaaProjektitInput {
   suunnittelustaVastaavaViranomainen: [Viranomainen!]
   vaihe: [Status!]
   vainProjektitMuokkausOikeuksin: Boolean
+  epaaktiivinen: Boolean
```

Comment:
Frontissa mun mielessä on nyt vähä karun näkösesti se epäaktiivisten checkkaaminen. En tekis uutta kenttää noille aktiivisille/epäktiivisille. Laittasin yhden kentän, jonka arvo vois olla Tie-, Rata-, Yleissuunnitelma tai Epäaktiivinen (tai undefined).

Kentän nimi tietty ei ehkä kantsis olla projektiTyyppi vaan joku muu esim. projektiKategoria.

Reply:
Tässä oli Joku Syy, miksi se tehtiin näin. Tyyliin indeksoinnin nopeus ja kätevät queryt. Suunniteltiin tää Mikon kanssa näin. Kelpaako sillä perusteella?

Reply:
Okei jos siinä joku tollanen oli